src/main/java/org/usfirst/frc4904/robot/subsystems/Indexer.java

```diff
@@ -0,0 +1,48 @@
+package org.usfirst.frc4904.robot.subsystems;
+
+import java.util.Set;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import org.usfirst.frc4904.standard.subsystems.SolenoidSubsystem;
+import org.usfirst.frc4904.standard.subsystems.SolenoidSubsystem.SolenoidState;
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
+
+/**
+ * Indexer - Wraps the indexer flippers and the run up belt.
+ * 
+ */
+public class Indexer {
+  public static final double DEFAULT_LIFT_SPEED = 0.0; // TODO: Untested value
+  public static final double DEFAULT_OFF_SPEED = 0.0;
+
+  public final SolenoidSubsystem flippers;
+  public final Motor liftBelts;
+
+  /**
+   * Indexer - Wraps the indexer flippers and the run up belt.
```

Comment:
You use the word "run up belt", even though the run up belt isn't (and shouldn't be) in this class.